src/index.js

```diff
@@ -70,7 +70,12 @@ function addPackageData (inputs, path) {
 function computeHash (pepper, paths, seed) {
   const inputs = []
   if (pepper) inputs.push(pepper)
-  if (seed) inputs.push(seed)
+  if (seed) {
```

Comment:
👍 - I'm just more accustomed to seeing that. 

I get `seed` for random number generators (you put it in at the beginning, to get things started - planting a seed). Salt is something you add (during cooking or afterwards) to change the flavor profile. This is just how I think of it in my head.